README.md

```diff
@@ -107,11 +107,23 @@ $ chmod +x fib
 $ qemu-arm fib
 ```
 
+### IR Regression Tests
+
 Verify that the emitted IRs are identical to the snapshots by specifying `check-snapshots` target when invoking `make`:
 ```shell
 $ make check-snapshots
 ```
 
+If the IR emission algorithm has been updated, you can update snapshots by specifying `update-snapshots` target when invoking `make`:
```

Comment:
@vacantron, Can you prepare some statements about the conditions to update snapshots?

Reply:
Done. @ChAoSUnItY , please cherry-pick vacantron/shecc@0d8d5e5fa683226763c202170b6d4cb81779aa44 .